test/unit/presenters/calendar_content_item_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+require "govuk-content-schema-test-helpers/test_unit"
+
+class CalendarContentItemTest < ActiveSupport::TestCase
```

Comment:
Should this file (introduced in commit 770219ab194d2d61e99c00b728ed5c0b79a319d4) have more tests?  It's missing `it "contains the correct locale"` and `context "localised content items"`